media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
```

Comment:
Yep, we generally don't support IE < 8 unless it's a specific page, but my comment was more about general consistency than browser support.